tests/libs/util/socket_helper.h

```diff
@@ -50,13 +78,26 @@ typedef class _base_socket
 typedef class _sender_socket : public _base_socket
```

Comment:
Now that you have changed the test network exchanges to be request-response, rather than pure send/receive; I would recommend renaming these classes to `client_socket` / `initiator_socket` and `server_socket` / `responder_socket` instead.

Reply:
Renamed to `client_socket` and `server_socket`